go/ql/lib/semmle/go/frameworks/K8sIoApimachineryPkgRuntime.qll

```diff
@@ -129,15 +64,9 @@ module K8sIoApimachineryPkgRuntime {
       // The format is not fixed. It depends on the receiver.
       none()
     }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp = this.getAnInput() and outp = this.getOutput()
-    }
   }
 
-  private class EncoderEncode extends TaintTracking::FunctionModel, Method,
-    MarshalingFunction::Range
-  {
+  private class EncoderEncode extends MarshalingFunction::Range, Method {
     EncoderEncode() {
       this.implements(packagePath(), "Encoder", "Encode") or
       this.hasQualifiedName(packagePath(), "WithVersionEncoder", "Encode")
```

Comment:
I think you're missing the model for `WithVersionEncoder.Encode` (and I'm guessing the test too)

Reply:
This is in the same package as, and implements, `Encoder`